format-message.js

```diff
@@ -272,3 +272,46 @@ https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-report
 📌 Escriba 6 para noticias y prensa
 📌 Escriba 0 para volver al menú
 `;
+
+function formatNewsMessage(newsList, whoNumber) {
+  let msg = `*LATEST NEWS & PRESS*
+
+`;
+  if (newsList.items.length > 5) {
+    newsList.items.length = 5;
+  }
+  newsList.items.forEach(item => {
+    let description = item.contentSnippet;
+    if (description.length > 300) {
+      description = description.substring(0,description.indexOf(".", 100)+1)
```

Comment:
I think we run the risk of returning an empty string here if there are no `.` characters after the 100th character:
```
> a = "123456. 7890"
'123456. 7890'
> a.substring(0, a.indexOf(".", 5)+1)
'123456.'
> a.substring(0, a.indexOf(".", 7)+1)
''
```

Reply:
Yes, but I do feel that is rather unlikely. I was thinking of just truncating it to 200ish characters if there isn't a `.`
Does that sound reasonable or should I make it less?

Reply:
I've addressed this in my latest commit.